### PR TITLE
[KIECLOUD-221] upgrade datagrid and amq broker versions

### PR DESCRIPTION
**Pull request by errantepiphany**
https://issues.jboss.org/browse/KIECLOUD-221
[KIECLOUD-221] upgrade datagrid and amq broker versions

Signed-off-by: David Ward <dward@redhat.com>

### CODE REVIEW

**Comment by errantepiphany** [comment]
@tchughesiv Please wait to merge until @spolti and @jakubschwan has had a chance to also look at https://github.com/jboss-container-images/rhdm-7-openshift-image/pull/248 and https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/303 . Thanks!

File: config/envs/rhpam-authoring-ha.yaml [hunk 1]
```diff
@@ -577,4 +577,3 @@ others:
           port:
             targetPort: rest
 ## ES/AMQ END
-
```
Comment (bmozaffa) [inline config/envs/rhpam-authoring-ha.yaml]: Please add back the newline at the end of the file, its absence causes diff errors and the like on some systems.

File: config/jms/activemq-jms-config.yaml [hunk 2]
```diff
@@ -222,4 +222,4 @@ servers:
             name: "[[.KieName]]-amq-jolokia"
 #[[end]]
 ## RANGE ends
-## KIE Servers END
\ No newline at end of file
+## KIE Servers END
```
Comment (bmozaffa) [inline config/jms/activemq-jms-config.yaml]: Same here, with newline at end of file, if it's removed

File: config/envs/rhpam-authoring-ha.yaml (hunk 1 above)
Comment (errantepiphany) [reply config/envs/rhpam-authoring-ha.yaml]: I was able to fix this one.

File: config/jms/activemq-jms-config.yaml (hunk 2 above)
Comment (errantepiphany) [reply config/jms/activemq-jms-config.yaml]: @bmozaffa I've tried everything, and no matter what editor I use (vim, gedit, or even online with the github editor), I can't get a newline to persist at the end of this file. Why is it necessary to be there? And if so, what can I do to add it back?

Comment (bmozaffa) [reply config/jms/activemq-jms-config.yaml]: Weird, I just submitted a PR against your branch. If you merge it, you can then go back and squash it in and do a push -f?

Without a newline, in some envs you also get a warning when doing a git diff, even when the content is the same near the EOF

Comment (spolti) [reply config/jms/activemq-jms-config.yaml]: on IntelliJ I had to put 2 blank lines to count one.

**Pull request merged**